source/retryable-writes/tests/README.rst

```diff
@@ -137,15 +138,17 @@ Each YAML file has the following keys:
 
   - ``description``: The name of the test.
 
+  - ``clientOptions``: Parameters to pass to MongoClient().
+
   - ``failPoint``: Document describing options for configuring the
     ``onPrimaryTransactionalWrite`` fail point on the primary server. This
     document should be merged with the
     ``{ configureFailPoint: "onPrimaryTransactionalWrite" }`` command document.
 
   - ``operation``: Document describing the operation to be executed. The
```

Comment:
please document the newly added `options` field

Reply:
The `options` field above was only added to the command monitoring spec tests, which doesn't discuss the test format in its [README](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/tests/README.rst). That said, the retryable write [README](https://github.com/mongodb/specifications/blob/master/source/retryable-writes/tests/README.rst) also doesn't discuss this format, as it only explains as far down as `tests.operation.arguments` and says that those arguments should correspond to names in the CRUD spec.